### PR TITLE
Release/4.0.0 beta.3

### PR DESCRIPTION
**Pull request by bhupeshappfoster**
### Description:
[Provide a brief description of the purpose and goal of this pull request.]

### Related Issue(s):

- [Ad system](https://github.com/AcclaroInc/pm-craft-translations/issues/551)
- [Service page](https://github.com/AcclaroInc/pm-craft-translations/issues/552)
- [User registration](https://github.com/AcclaroInc/pm-craft-translations/issues/550)

### Changes Made:
[List the changes made in this pull request to address the issue or implement the feature.]

**Added:**

- 

**Changed:**

-

**Deprecated:**

-

**Removed:**

-

**Fixed:**

-

**Security:**

-

### Testing:
[Describe the testing performed to ensure the changes are functioning as expected.]

- 

### Quality Assurance (QA):

- [ ] The code has been reviewed and approved by the QA team.
- [ ] The changes have been tested on different environments (e.g., staging, production).
- [ ] Integration tests have been performed to verify the interactions between components.
- [ ] Performance tests have been conducted to ensure the changes do not impact system performance.
- [ ] Any necessary database migrations or data transformations have been executed successfully.
- [ ] Accessibility requirements have been considered and tested (e.g., screen reader compatibility, keyboard navigation).

### Resources:

**Screenshots (if applicable):**
[Include any relevant screenshots to visually demonstrate the changes.]

### Wrapping up

**Checklist:**

- [ ] The code builds without any errors or warnings.
- [ ] The code follows the project's coding conventions and style guidelines.
- [ ] Unit tests have been added or updated to cover the changes made.
- [ ] The documentation has been updated to reflect the changes (if applicable).
- [ ] All new and existing tests pass successfully.
- [ ] The PR has been reviewed by at least one other team member.

**Additional Notes:**
[Include any additional notes, considerations, or context that may be relevant.]

### CODE REVIEW

**Comment by bhupeshappfoster** [comment]
### :new: Updates

Branch deployed to dev server for final review.

**Comment by bhupeshappfoster** [comment]
@sidedwards 

Deployed this beta 3 changes branch on dev once approved we can have this beta3 version released & freeze our code for Stable release.

File: src/Constants.php [hunk 1]
```diff
@@ -379,4 +379,30 @@ class Constants
...
+
+    // Dashboard will be array of ads to show more ads if needed in future
+    const ADS_CONTENT = [
+        "dashboard" => [
+            [
+                "heading" => "AI/Machine Translation",
+                "content" => "Translate more content faster with AI.",
+                "linkText" => "Try it now",
+                "link" => self::CTA_AI_ML_TRANSLATION,
+            ]
```
Comment (sidedwards) [inline src/Constants.php:391]: @bhupeshappfoster is this excluded from the dashboard widgets?

File: src/assetbundles/src/css/services.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
+.card-box {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (sidedwards) [inline src/assetbundles/src/css/services.css:4]: @sharmarishabh1 can we change this to `align-items: flex-start;`

**Review by sidedwards — CHANGES_REQUESTED** [review]
@bhupeshappfoster when we release this, it should be v4.0.0.

File: src/assetbundles/src/css/services.css (hunk 2 above)
Comment (sharmarishabh1) [reply src/assetbundles/src/css/services.css:4]: Fixed here #530 

@sidedwards @bhupeshappfoster 
Please review the same whenever possible.

**Pull request closed without merging**